src/app/features/public-form/edit-prefilled-values/edit-prefilled-values.component.ts

```diff
@@ -0,0 +1,183 @@
+import { CommonModule } from "@angular/common";
+import { Component, OnInit, inject } from "@angular/core";
+import {
+  FormArray,
+  FormBuilder,
+  ReactiveFormsModule,
+  Validators,
+} from "@angular/forms";
+import { DefaultValueOptionsComponent } from "app/core/admin/admin-entity-details/admin-entity-field/default-value-options/default-value-options.component";
+import { FieldGroup } from "app/core/entity-details/form/field-group";
+import { EntityRegistry } from "app/core/entity/database-entity.decorator";
+import { EditComponent } from "app/core/entity/default-datatype/edit-component";
+import { EntityConstructor } from "app/core/entity/model/entity";
+import { MatFormFieldModule } from "@angular/material/form-field";
+import { MatSelectModule } from "@angular/material/select";
+import { FontAwesomeModule } from "@fortawesome/angular-fontawesome";
+import { FormConfig } from "app/core/entity-details/form/form.component";
+import { DefaultValueConfig } from "app/core/entity/schema/default-value-config";
+import { HelpButtonComponent } from "app/core/common-components/help-button/help-button.component";
+import { MatButtonModule } from "@angular/material/button";
+import { MatTooltipModule } from "@angular/material/tooltip";
+import { FormFieldConfig } from "app/core/common-components/entity-form/FormConfig";
+import { EntityFieldSelectComponent } from "app/core/entity/entity-field-select/entity-field-select.component";
+
+@Component({
+  selector: "app-edit-prefilled-values",
+  standalone: true,
+  imports: [
+    DefaultValueOptionsComponent,
+    CommonModule,
+    ReactiveFormsModule,
+    MatFormFieldModule,
+    MatSelectModule,
+    MatTooltipModule,
+    HelpButtonComponent,
+    FontAwesomeModule,
+    MatButtonModule,
+    EntityFieldSelectComponent,
+  ],
+  templateUrl: "./edit-prefilled-values.component.html",
+  styleUrls: ["./edit-prefilled-values.component.scss"],
+})
+export class EditPrefilledValuesComponent
+  extends EditComponent<FieldGroup[]>
+  implements OnInit
+{
+  entityConstructor: EntityConstructor;
+  formConfig: FormConfig;
+  defaultValue: DefaultValueConfig;
+  prefilledValueSettings = this.fb.group({
+    prefilledvalue: this.fb.array([]),
+  });
+
+  private entities = inject(EntityRegistry);
+
+  constructor(private fb: FormBuilder) {
```

Comment:
can probably do `private fb = inject(FormBuilder)` to avoid having to call super constructor?